docs/installation.rst

```diff
@@ -0,0 +1,236 @@
+
+    pip3 install ansible-navigator --user
+
+#. Add the installation path to the PATH:
+
+   .. code-block:: console
+
+    echo 'export PATH=$HOME/Library/Python/3.8/bin:$PATH' >> ~/.zshrc
+
+#. Open a new terminal or refresh the PATH:
+
+   .. code-block:: console
+
+    source ~/.zshrc
+
+#. Launch ansible-navigator:
+
+   .. code-block:: console
+
+    ansible-navigator
+
+#. ``ansible-navigator`` triggers a one-time download of the demo execution-environment image.
+
+
+Windows with WSL2
+*****************
+
+Requirements
+============
+
+* Windows Subsystem for Linux 2
+* Either ``podman`` or ``Docker Desktop for Windows``
+* Internet access (during initial installation)
+
+Setup WSL2 for Windows with Ubuntu
+==================================
+
+#. Install `WSL 2 <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__.
+#. Install the Ubuntu 20.04 LTS Linux distribution from the Microsoft store.
+#. Open PowerShell and set the default WSL 2 distribution:
+
+   .. code-block:: console
+
+    wsl --set-default ubuntu
+
+#. Launch the Ubuntu virtual machine from the Windows menu and complete the initial Ubuntu set-up.
+#. From the Ubuntu terminal, create the ``/dev/mqueue`` directory:
+
+   .. code-block:: console
+
+    sudo mkdir /dev/mqueue
+
+
+Install the desired container engine for execution environment support
+======================================================================
+
+* Installation instructions for ``podman`` on Ubuntu 20.04 LTS.
+
+   .. note::
+
+      The podman package is available in the official repositories for Ubuntu 20.10 and newer.
+      Since interim releases of Ubuntu are not available on the Microsoft Store for WSL the
+      `Kubic project <https://build.opensuse.org/package/show/devel:kubic:libcontainers:stable/podman>`__ package can be used.
+
+   #. Update the ubuntu package index:
+
+      .. code-block:: console
+
+       sudo apt update
+
+   #. Install system dependencies for ``podman``:
+
+      .. code-block:: console
+
+       apt-get install curl wget gnupg2
+
+   #. Source the Ubuntu release:
+
+      .. code-block:: console
+
+       source /etc/os-release
+
+   #. Add the ``podman`` repository:
+
+     .. code-block:: console
+
+      sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
+
+   #. Download the GPG key:
+
+     .. code-block:: console
+
+      wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_${VERSION_ID}/Release.key -O- | sudo apt-key add -
+
+   #. Update using the new repository:
+
+      .. code-block:: console
+
+       sudo apt-get update
+
+   #. Install ``podman``:
+
+      .. code-block:: console
+
+       sudo apt-get install podman
+
+* Follow the `Docker Desktop for Windows <https://hub.docker.com/editions/community/docker-ce-desktop-windows>`__ installation instructions.
```

Comment:
I'd preface this because if someone isn't paying close attention, it seems like it's part of the above (i.e., do all this stuff to get podman working, and then follow this guide to install docker).

Maybe something like "If you choose not to use `podman` as above, you can alternatively install Docker by following these instructions..."